training/training.py

```diff
@@ -285,11 +291,12 @@ def main():
     parser = ArgumentParser()
     parser.add_argument("-d", "--data_dir", dest="data_dir", type=str, required=True)
     parser.add_argument('-c', '--command_list', nargs='+', dest="command_list", type=str, required=True)
-    parser.add_argument('-e', '--num_epochs', dest="num_epochs", type=int, default=500)
-    parser.add_argument('-f', '--training_log_frequency', dest="training_log_frequency", type=int, default=50)
+    parser.add_argument('-e', '--num_epochs', dest="num_epochs", type=int, default=26)
+    parser.add_argument('-f', '--training_log_frequency', dest="training_log_frequency", type=int, default=5)
     parser.add_argument('-l', '--log_file', dest="log_file")
-    parser.add_argument('-b', '--batch_size', dest="batch_size", type=int, default=100)
-    parser.add_argument('-lr', '--learning_rate', dest="learning_rate", type=float, default=0.01)
+    parser.add_argument('-b', '--batch_size', dest="batch_size", type=int, default=64)
+    parser.add_argument('-rs', '--random_seed', dest="random_seed", type=int, default=10)
```

Comment:
In these cases, the argument name is already mapped to the appropriate `dest`, e.g., `--batch_size` --> `parser.parse_args().batch_size`